LayoutKit/Alignment.swift

```diff
@@ -117,11 +117,11 @@ public struct Alignment {
     }
 
     /// A function that aligns size in rect.
-    public typealias Aligner = (size: CGSize, rect: CGRect) -> CGRect
+    public typealias Aligner = (_ size: CGSize, _ rect: CGRect) -> CGRect
```

Comment:
I don't see an obvious way to keep them. The alternative is probably to just declare a protocol. Seems like overkill here though.

```
Alignment.swift:120:33: Function types cannot have argument label 'size'; use '_' instead
```